model/project_parser.go

```diff
@@ -116,6 +116,7 @@ type parserTask struct {
 	DependsOn       parserDependencies  `yaml:"depends_on,omitempty" bson:"depends_on,omitempty"`
 	Commands        []PluginCommandConf `yaml:"commands,omitempty" bson:"commands,omitempty"`
 	Tags            parserStringSlice   `yaml:"tags,omitempty" bson:"tags,omitempty"`
+	RunOn           parserStringSlice   `yaml:"run_on,omitempty" bson:"run_on,omitempty"`
```

Comment:
could we be consistent about the naming of this vs. the corresponding field in the project? that one is distros but this is run_on

Reply:
That's true, but I chose RunOn to be consistent with the other parser project structures

Reply:
In that case can the one in the project be named RunOn? Distros is supposed to be deprecated anyway